COPYRIGHT

```diff
@@ -43,11 +43,24 @@ COMPLETENESS, OR USEFULNESS OR ANY INFORMATION, APPARATUS, PRODUCT, OR
 PROCESS DISCLOSED, OR REPRESENTS THAT ITS USE WOULD NOT INFRINGE PRIVATELY
 OWNED RIGHTS.
 
-© 2017 Los Alamos National Security, LLC
-This material was produced under U.S. Government contract DE-AC52-06NA25396
-for Los Alamos National Laboratory (LANL), which is operated by
-Los Alamos National Security, LLC for the U.S. Department of Energy.
-The U.S. Government has rights to use, reproduce, and distribute this software.
-NEITHER THE GOVERNMENT NOR LOS ALAMOS NATIONAL SECURITY, LLC
-MAKES ANY WARRANTY, EXPRESS OR IMPLIED, OR ASSUMES ANY LIABILITY
-FOR THE USE OF THIS SOFTWARE.
+
+© 2017 - 2018 Triad National Security, LLC
+ALL RIGHTS RESERVED
+
+This software was produced under U.S. Government contract 89233218CNA000001 for
+Los Alamos National Laboratory (LANL), which is operated by Triad National
+Security, LLC for the U.S. Department of Energy/National Nuclear Security
+Administration. The U.S. Government has rights to use, reproduce, and distribute
+this software. NEITHER THE GOVERNMENT NOR TRIAD NATIONAL SECURITY, LLC MAKES ANY
+WARRANTY, EXPRESS OR IMPLIED, OR ASSUMES ANY LIABILITY FOR THE USE OF THIS
+SOFTWARE. If software is modified to produce derivative works, such modified
+software should be clearly marked, so as not to confuse it with the version
+available from LANL.
+
+Additionally, this library is free software; you can redistribute it and/or
```

Comment:
This is already covered by the `LICENSE` file, isn't it?

Reply:
You're right, I removed this paragraph.